app/src/main/java/com/kickstarter/ui/activities/CreatorDashboardActivity.java

```diff
@@ -0,0 +1,106 @@
+package com.kickstarter.ui.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.widget.TextView;
+
+import com.kickstarter.R;
+import com.kickstarter.libs.BaseActivity;
+import com.kickstarter.libs.KSCurrency;
+import com.kickstarter.libs.KSString;
+import com.kickstarter.libs.RefTag;
+import com.kickstarter.libs.qualifiers.RequiresActivityViewModel;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.ui.IntentKey;
+import com.kickstarter.viewmodels.CreatorDashboardViewModel;
+
+import java.math.RoundingMode;
+
+import butterknife.Bind;
+import butterknife.BindString;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.observeForUI;
+
+@RequiresActivityViewModel(CreatorDashboardViewModel.ViewModel.class)
+public final class CreatorDashboardActivity extends BaseActivity<CreatorDashboardViewModel.ViewModel> {
+
+  protected @Bind(R.id.creator_dashboard_amount_raised) TextView amountRaisedTextView;
+  protected @Bind(R.id.creator_dashboard_funding_text) TextView fundingTextTextView;
+  protected @Bind(R.id.creator_dashboard_backer_count) TextView backerCountTextView;
+  protected @Bind(R.id.creator_dashboard_project_name) TextView projectNameTextView;
+  protected @Bind(R.id.creator_dashboard_time_remaining) TextView timeRemainingTextView;
+  protected @Bind(R.id.creator_dashboard_time_remaining_text) TextView timeRemainingTextTextView;
+  protected @BindString(R.string.discovery_baseball_card_stats_pledged_of_goal) String pledgedOfGoalString;
+
+  private KSCurrency ksCurrency;
+  private KSString ksString;
+
+  @Override
+  protected void onCreate(final @Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.creator_dashboard_layout);
+    ButterKnife.bind(this);
+
+    this.ksCurrency = this.environment().ksCurrency();
+    this.ksString = this.environment().ksString();
+
+    viewModel.outputs.latestProject()
```

Comment:
If we're going to use explicit `this`, can we create a separate PR introducing http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.html and audit our entire codebase for infractions? If we try to maintain this without an automated tool we're going to miss a lot of stuff.